cfscrape/api.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+from .scraper import CloudflareScraper
+
+
+scraper = None
```

Comment:
If preferable, we could create the scraper when imported instead of deferring until the API is actually used.

Reply:
i.e. The entire contents of this file could be replaced with:
```py
# -*- coding: utf-8 -*-

import sys
from .scraper import CloudflareScraper


sys.modules[__name__] = CloudflareScraper()
```